docs/reference/commandline/swarm_init.md

```diff
@@ -84,11 +84,18 @@ The URL specifies the endpoint where signing requests should be submitted.
 
 ### `--force-new-cluster`
 
-This flag forces an existing node that was part of a quorum that was lost to restart as a single node Manager without losing its data
+This flag forces an existing node that was part of a quorum that was lost to restart as a single node Manager without losing its data.
 
 ### `--listen-addr value`
 
-The node listens for inbound Swarm manager traffic on this IP:PORT
+The node listens for inbound Swarm manager traffic on this IP:PORT. If unspecified, Docker
+will check if the system has a single IP address, and use that IP address with with the
+default port 2377. If the system has multiple IP addresses, `--listen-addr` must be
+specified so that the correct address is chosen for inter-manager communication and overlay
+networking.
+
+It is also possible to specify a network interface to listen on that interface's address;
+for example `--listen-addr eth0:2377`.
```

Comment:
It is optional. If you don't specify it, `:2377` is implied. But I think specifying a port is good practice. Should I clarify this?